sulu/article-bundle/2.0/config/packages/sulu_article.yaml

```diff
@@ -0,0 +1,16 @@
+sulu_article:
+    index_name: '%env(resolve:ELASTICSEARCH_INDEX)%'
+    hosts:
+        - '%env(resolve:ELASTICSEARCH_HOST)%'
```

Comment:
currently the bundle does not work without configuring a type mapping we should add here:

```
sulu_article:
    types:
        article:
            translation_key: "app.article"
```